apps/hash-external-services/turbo.json

```diff
@@ -31,6 +31,15 @@
         "codegen"
       ]
     },
+    "deploy:prod": {
+      "cache": false,
+      "dependsOn": [
+        "@apps/hash-graph#build:docker:prod",
+        "@apps/hash-agents#build:docker",
+        "@apps/hash-ai-worker-ts#build:docker",
```

Comment:
We don't use the TS workers in the production image :)
```suggestion
```

Reply:
Good flag, thanks 😄